mule-extensions-api/src/main/java/org/mule/runtime/extension/internal/dsl/syntax/XmlDslSyntaxResolver.java

```diff
@@ -129,20 +134,24 @@ public XmlDslSyntaxResolver(ExtensionModel model, ImportTypesStrategy importType
    */
   public DslElementSyntax resolve(final NamedObject component) {
```

Comment:
can't do this now, it's heavily used by Studio to resolve everything dsl-related